### DISCUSSION TITLE
Help with Copilot and VSCode

### DISCUSSION THREAD

**Discussion by RakhaSatriaPratama** [community/community#41951] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I have registered copilot, I have installed the copilot extension on vsCode, I have connected vsCode to github. but why can't I run github copilot and no icon appears at the bottom of my vsCode, can anyone help me?

**Comment by iamcjwilliams** [comment]
@RakhaSatriaPratama Sorry for the trouble here, please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-visual-studio-code
